accepted/00075-live-patching-enhancements.md

```diff
@@ -0,0 +1,170 @@
+
+Below are some examples of such sets of filters:
+
+Live patching project with a single filter to exclude regular kernel packages newer than a specific version:
+ - `DENY erratum contains package with epoch/version/release greater than kernel-default <EVR> (package_nevr)`
+
+Live patching project using `ALLOW` overrides:
+ - `DENY errata containing "update for the Linux Kernel" in synopsis`
+ - `ALLOW errata containing "Live Patch" in synopsis` *- an override to the first rule*
+ - `ALLOW a specific patch containing current kernel version (via advisory name)` *- an override to the first rule*
+
+### CLM project wizards
+
+The process of live patching can be made more user-friendly using "CLM project wizards". The wizards are alternatives to the regular filter creation dialogs that provide a tailored interaction to create multiple filters that serve a single purpose. These dialogs can be accessed via the "Use wizard" button in the filter creation dialog. This concept can be extended to other use cases and workflows in the future.
+
+When completed, the wizards will create multiple filters that look like the examples mentioned in the section above to set up a project with a specific purpose. Since live patching channels can be composed with the existing filter types, no additional filter type needs to be implemented.
+
+An advantage of this approach is that since the workflow is defined with a set of regular filters, advanced users can modify these filters to achieve finer control over the project.
+
+A wizard can be applied on a project at any time, adding filters as described. Multiple wizards can be applied to a single project and it's the user's responsibility to make sure the created filters do not conflict with each other.
+
+A wizard can be applied independently from a project. In that case, the filters will be created without being added to a specific project. These filters can be used in any project later on.
+
+In scope of this RFC, two different wizards are proposed. The process of creating the filters using these wizards are described below.
+
+#### Live patching for a system
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a SLES client from a combobox
+ 3. Select a kernel version from a dropdown (current kernel version of the system is preselected)
+
+The resulting filters will set up the project for live patching from the specified kernel version.
+If the wizard is accessed from the project view, all the created filters will be automatically added to the project.
+
+#### Live patching for a product
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a synced SLES product from a dropdown
```

Comment:
I think `2.` is not needed. When you select a Project, you have already channels in that project which defines which kernel versions are available. So we can just show what is available.

Or is the idea to add also channels? But than we talk about a CLM project wizard and not about a Filter Wizard.

Reply:
Similar to what we experienced with AppStream filters, the order in which the user adds different entities is not enforced, so the user may start with adding filters before channels or anything else. Moreover, the design allows the wizard to be accessed from the filters page independent from any project.